src/middlewared/middlewared/pytest/unit/plugins/crypto/certs.py

```diff
@@ -207,3 +211,192 @@ def test__generating_cert(generate_params, key_type, key_size, cert_info):
             cert_details[k].pop('SubjectKeyIdentifier')
 
         assert cert_info[k] == cert_details[k], cert_details
+
+
+@pytest.mark.parametrize('data', [
```

Comment:
Please use `id=` kwarg to this parametrize decorator. Otherwise, the console from pytest will be polluted with the key contents.